src/main/resources/multiverse-core_en.properties

```diff
@@ -19,6 +19,17 @@ mv-core.clone.success=Cloned world '{world}'!
 # /mv confirm
 mv-core.confirm.description=Confirms dangerous commands before executing them.
 
+# /mv coordinates
+mv-core.coordinates.description=Simply sends your coordinates
+mv-core.coordinates.error.players-only=&cThis command can only be used by players
```

Comment:
Having a `-` in the message path may not work, did you test it?

Reply:
Huh, no it doesn’t work but I’m sure I committed a fix for that 🤔

Reply:
It's not supposed to work, I think so away with the - for now